kaztron/cog/dice.py

```diff
@@ -30,6 +30,8 @@ def __init__(self, bot):
             raise ValueError("Channel {} not found".format(self.config.get('dice', 'channel_dice')))
         await super().on_ready()
 
+    max_choices = 20
```

Comment:
class vars/constants should be at the top of the class definition, before `__init__`, and probably also named in allcaps